azure/resource-containers/pf-management-group/functions/private/new-pfazmanagementgroup.ps1

```diff
@@ -0,0 +1,15 @@
+Function New-PfAzManagementGroup{
```

Comment:
This is the private function now that will use the az module to Create the management group in Azure